evasquez26/angular4-startup
# S3 One 39 40 enroll and unenroll to a class

## Story / Task

[ONE-39](https://bixlabs.atlassian.net/browse/ONE-39) and [ONE-40](https://bixlabs.atlassian.net/browse/ONE-40)

## What this PR does?



### Before 

- We could not enroll and unenroll from courses

### After

- We can enroll or unenroll from courses if the course has classes created

### How to test / reproduce

- Go to create a class to a course
- Assign an enrolldate that is one day from now
- Go to the course details (tomorrow :laughing:)
- Observe that there's a button to enroll to the course
- Select a start date of a class
- Click the enroll button
- You should see a success message that you've been enrolled successfully to the course
- Observe that there's now an unenroll button
- Clicking the unenroll button should let you unenroll from the course

### TODOs
- [x] Rebase after merging #29 

### Related PRs

N/A

## Migrations

NO
